libs/sql-parser/src/main/java/io/crate/sql/SqlFormatter.java

```diff
@@ -152,12 +152,20 @@ private SqlFormatter() {
     }
 
     public static String formatSql(Node root) {
-        return formatSql(root, null);
+        return formatSql(root, null, false);
+    }
+
+    public static String formatSqlInline(Node root) {
```

Comment:
What's the inline formatting version for and how is this different from the other format function?

Should probably have some docs - but would be even better if we could avoid this. Makes it more difficult to reason about in which situation which should be used.

Also looks like quite a few things here don't have coverage?

Reply:
The inline format won't add line breaks and indents. I've added this to make messages of the ColumnUnknownError more readable for expressions, having line breaks and indents may not so nice.
Missing coverage is probably due to my introduced randomization, will remove that as suggested.

Reply:
Ok makes sense - some docs explaining that difference would be good.

And could consider internally parameterizing the newline character + indent space. If newline is then a `` and indent always `0` that could avoid some of the `if (inline)` branches